covid_xprize/standard_predictor/xprize_predictor.py

```diff
@@ -109,10 +107,15 @@ def predict(self,
                     "Date": [],
                     "PredictedDailyNewCases": []}
 
+        # Fix for past predictions
+        geos = npis_df.GeoID.unique()
+        truncated_df = self.df[self.df.Date < start_date_str]
+        country_samples = self._create_country_samples(truncated_df, geos, False)
```

Comment:
Instead we recreate `country_samples` when `predict` is called, according to the Oxford DataFrame truncated to before the start date.